Assets/Scripts/Structures/Select.cs

```diff
@@ -267,6 +273,17 @@ private void DisplayEffects()
                 badges[i].background.color = Colors.StatColours[effects[i].Key];
                 badges[i].icon.sprite = statIcons[effects[i].Key];
             }
+            
+            if (!SelectedStructure.Bonus.HasValue) return;
+            bonusBadge.transform.localPosition = new Vector2(0, -320);
+            StartCoroutine(Algorithms.DelayCall(0.2f, () =>
+            {
+                bonusBadge.icon.sprite = statIcons[SelectedStructure.Bonus.Value];
+                bonusBadge.background.color = Colors.StatColours[SelectedStructure.Bonus.Value];
+                bonusText.text = SelectedStructure.Blueprint.adjacencyConfig.Description;
+                bonusBadge.transform.DOLocalMove(new Vector2(0, -270), 1f);
+                bonusBadge.canvasGroup.DOFade(1, 1f);
+            }));
```

Comment:
This can be done using DoTween itself with the following:

```suggestion
            
            if (!SelectedStructure.Bonus.HasValue) return;
            bonusBadge.transform.localPosition = new Vector2(0, -320);
            
            bonusBadge.canvasGroup.DOFade(1, 1f);
            bonusBadge.transform
                .DOLocalMove(new Vector2(0, -270), 1f)
                .SetDelay(0.2f)
                .OnStart(() => {
                    bonusBadge.icon.sprite = statIcons[SelectedStructure.Bonus.Value];
                    bonusBadge.background.color = Colors.StatColours[SelectedStructure.Bonus.Value];
                    bonusText.text = SelectedStructure.Blueprint.adjacencyConfig.Description;
                });
```